src/DotVVM.Framework/Compilation/Styles/StyleBuilderMethods.Helpers.cs

```diff
@@ -0,0 +1,23 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using DotVVM.Framework.Utils;
+using System.Reflection;
+using System.Linq;
+using DotVVM.Framework.Controls;
+using DotVVM.Framework.Compilation.Styles;
+
+public static partial class StyleBuilderExtensionMethods
```

Comment:
Is there a reason why these two methods are in a separate file?

Reply:
These methods are specific for some property or control, the other file is for general purpose functions. I think we will add more of them in this file.